src/types.ts

```diff
@@ -99,6 +99,29 @@ export interface VitePWAOptions {
    * option directory.
    */
   includeManifestIcons: true
+  /**
+   * Configuration for `networkFirst` strategy.
+   * Keep it here if in future we need to extend its configuration.
+   */
+  networkFirst?: {
+    /**
+     * Should we use `Custom Cache Network Race Strategy` or `Network First Cache Strategy`?
+     * @default true
+     */
+    raceStrategy?: boolean
```

Comment:
I think we should change to `Network First Cache Strategy` by default

Reply:
Since `Custom Cache Network Race Strategy` is for special case (when using smalls chunks) I'll keep there but setting  `Network First Cache Strategy` as default.

I'll change the `jsdoc` on `types.ts`, the `README.md` and modify the logic a little bit.